packaging/suse/datadog-agent.init

```diff
@@ -417,7 +418,12 @@ case "$1" in
         DOGSTATSD_RETURN=$?
         su $AGENTUSER -c "$FORWARDERPATH info"
         FORWARDER_RETURN=$?
-        exit $(($COLLECTOR_RETURN+$DOGSTATSD_RETURN+$FORWARDER_RETURN))
+        TRACEAGENT_RETURN=0
+        if [ -x $TRACEAGENTPATH ];
+           su $AGENTUSER -c "$TRACEAGENTPATH -info"
+           TRACEAGENT_RETURN=$?
+        fi
+        exit $(($COLLECTOR_RETURN+$DOGSTATSD_RETURN+$FORWARDER_RETURN+$TRACEAGENT_RETURN))
```

Comment:
as mentioned in chat, i'd prefer if we don't include $TRACEAGENT_RETURN in the exit code. we mark it as an optional process, disabled by default. since `info` is sometimes used as a healthcheck - i think we should consider the agent "healthy" even when `trace-agent -info` exits non-zero. @olivielpeau any thoughts on this?